spec/lib/root_route_constraint_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'spec_helper'
+
+describe RootRouteConstraint do
+  describe '#matches?' do
+
+    it 'returns false, if no conference is live' do
+      constraint = RootRouteConstraint.new
+      expect(constraint.matches?).to eq false
+    end
+
+    it 'returns false, when one conference is live but no splashpage' do
+      create(:conference)
+      constraint = RootRouteConstraint.new
+      expect(constraint.matches?).to eq false
+    end
+
+    it 'returns false, when one conference is live but no public splashpage' do
+      conference = create(:full_conference)
+      conference.splashpage.public = false
```

Comment:
Also I think there is some problem because all the spec in `root_route_constraint_spec.rb` are returning `flase` whereas the spec `return true, when one conference is live and has public splashpage`  must return `true`